### PR TITLE
GEOPY-883: more robust dev, install and run scripts

### PR DESCRIPTION
**Pull request by sebhmg**
**GEOPY-883 - conda not found at install time**

### CODE REVIEW

**Review by andrewg-mira — APPROVED** [review]

File: Start_applications.bat [hunk 1]
```diff
@@ -7,6 +7,8 @@ if !errorlevel! neq 0 (
   exit /B !errorlevel!
 )
 
+set ENV_NAME=geoapps
+
 set MY_CONDA=!MY_CONDA_EXE:"=!
-call "!MY_CONDA!" activate geoapps && jupyter notebook geoapps\index.ipynb
-cmd /k
+call "!MY_CONDA!" run -n %ENV_NAME% jupyter notebook geoapps\index.ipynb
+cmd /k "!MY_CONDA!" activate %ENV_NAME%
```
Comment (domfournier) [inline Start_applications.bat:14]: Same comment as on GEOPY-891, kind of weird that you activate after.

Comment (domfournier) [reply Start_applications.bat:14]: Does that work?

Comment (sebhmg) [reply Start_applications.bat:14]: yes it does work. Actually, if jupyter starts, it never reaches that point.
If juyter fails, then it places the user in a terminal where the environment is activated. Possibly useful for diagnostic

Comment (domfournier) [reply Start_applications.bat:14]: I see. Great.

**Review by domfournier — APPROVED** [review]

**Pull request merged**